src/poliastro/bodies.py

```diff
@@ -201,6 +203,7 @@ def plot(
     R_mean=constants.R_mean_earth,
     R_polar=constants.R_polar_earth,
     rotational_period=constants.rotational_period_earth,
+    solar_rotational_period=constants.solar_rotational_period_earth,
```

Comment:
I wonder if there is a way to derive the solar rotational period given the rotational period and the orbital period around the Sun. Am I missing something?

Reply:
Maybe it's possible, but I don't have the right knowledge to do so. The paper sets that variable as a constant.

Reply:
Doing some research I found [here](https://books.google.com/books?id=pp3wDwAAQBAJ&pg=PA19#v=onepage&q&f=false) the relation between Psol, Psid and the Period of the attractor around the sun
![image](https://user-images.githubusercontent.com/9794029/89772196-fdda7e00-dad7-11ea-8d28-721cddd1accc.png)
And Period of the attractor around the sun you can obtain it from
![image](https://user-images.githubusercontent.com/9794029/89772402-527df900-dad8-11ea-8d5c-58620d10ef99.png)
detailed in the book An introduction to modern astrophysics. 2nd edition. Pearson
So there is no need to add this as a constant